docs/proposals/rollout/rollout.md

```diff
@@ -300,8 +288,39 @@ type TrafficRoutingConfig struct {
     // +optional
     CorsPolicy *istiov1alpha3.CorsPolicy `json:"corsPolicy,omitempty"`
 
+    // RolloutAnalysisCheckTimes defines the number of traffic analysis checks to run for A/B Testing and Blue/Green
```

Comment:
Please check all `Blue/Green` and A/B, usually it is with A/B testing ·Blue/Green Deployment